porcupine/plugins/directory_tree.py

```diff
@@ -229,7 +235,8 @@ def project_has_open_filetabs(self, project_id: str) -> bool:
 
     def _hide_old_projects(self, junk: object = None) -> None:
         for project_id in self.get_children(""):
-            if not get_path(project_id).is_dir():
+            project_path = get_path(project_id)
+            if project_path is None or not project_path.is_dir():
```

Comment:
It cannot be `None`.

```suggestion
            assert project_path is not None
            if not project_path.is_dir():
```